CONTRIBUTING.md

```diff
@@ -65,6 +92,14 @@ cd path/to/some-other-project
 snowpack dev --verbose --reload
 ```
 
+To test a local version of the `create-snowpack-app` templates use
+
+NOTE This does not work, trying to figure this one out
```

Comment:
one thing that could help: splitting this into two different snippets:

To test CLI changes: `node /path/to/snowpack/create-snowpack-app/cli [my-new-dir] --template @snowpack/app-template-vue`


To test a new template: `npx create-snowpack-app [my-new-dir] --template /path/to/template`

Reply:
good idea 👍🏼 

Let me know if I can help @melissamcewen

Reply:
I still get `[ERROR] Unable to find "/Users/melissamcewen/sites/snowpack/create-snowpack-app/app-template-vue" in the npm registry.` with either